command/texttools.js

```diff
@@ -22,7 +22,7 @@ var texttools = {
 
   cleanText: function(htmlStr) {
     // Remove the HTML marks.
-    var strippedHtml = htmlStr.replace(/<[^>]+>/g, '');
+    var strippedHtml = htmlStr.replace(/<[^>]+>/g, ' ');
```

Comment:
Are you sure about this change?  Was this a code formatting from the prettifier or an actual change to fix a bug.  Thanks.